doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1364 @@
+      * :ref:`ug_nrf54h20_suit_components` - An explanation of SUIT components, found within the manifest(s).
+      * :ref:`ug_nrf54h20_suit_hierarchical_manifests` - An explanation of the SUIT manifest topology that Nordic Semiconductor has implemented for the nRF54H20 SoC.
+      * :ref:`ug_nrf54h20_suit_compare_other_dfu` - A comparison of SUIT with other bootloader and DFU procedures supported in the |NCS|.
+
+    The following user guides:
+
+      * :ref:`ug_nrf54h20_suit_customize_dfu` - Describing how to customize the SUIT DFU procedure (and a quick-start guide version :ref:`ug_nrf54h20_suit_customize_dfu_qsg`).
+      * :ref:`ug_nrf54h20_suit_fetch` - Describing how to reconfigure an application that uses the push model to a fetch model-based upgrade.
+      * :ref:`ug_nrf54h20_suit_external_memory` - Describing how to enable external flash memory when using SUIT.
+
+  * :ref:`ug_wifi_mem_req_raw_mode` under :ref:`ug_wifi_advanced_mode`.
+  * The :ref:`bt_mesh_models_common_blocking_api_rule` section to the :ref:`bt_mesh_models_overview` page.
+  * Steps for nRF54 devices across all supported samples to reflect the new button and LED numbering on the nRF54H20 DK and the nRF54L15 PDK.
+  * The :ref:`test_framework` section for gathering information about unit tests, with a new page about :ref:`running_unit_tests`.
+  * List of :ref:`debugging_tools` on the :ref:`debugging` page.
+  * Recommendation for the use of a :file:`VERSION` file for :ref:`ug_fw_update_image_versions_mcuboot` in the :ref:`ug_fw_update_image_versions` user guide.
+
+* Updated:
+
+  * The :ref:`cmake_options` section on the :ref:`configuring_cmake` page with the list of most common CMake options and more information about how to provide them.
+  * The table listing the :ref:`boards included in sdk-zephyr <app_boards_names_zephyr>` with the nRF54L15 PDK and nRF54H20 DK boards.
+  * The :ref:`ug_wifi_overview` page by separating the information about Wi-Fi certification into its own :ref:`ug_wifi_certification` page under :ref:`ug_wifi`.
+  * The :ref:`ug_bt_mesh_configuring` page with an example of possible entries in the Settings NVS name cache.
+  * The :ref:`lib_security` page to include all security-related libraries.
+  * The trusted storage support table in the :ref:`trusted_storage_in_ncs` section by adding nRF52833 and replacing nRF9160 with nRF91 Series.
+  * The :ref:`ug_nrf52_developing` and :ref:`ug_nrf5340` by adding notes about how to perform FOTA updates with samples using random HCI identities, some specifically relevant when using the iOS app.
+  * Improved the :ref:`ug_radio_fem` user guide to be up-to-date and more informative.
+  * The :ref:`bt_fast_pair_readme` page to document support for the FMDN extension and aligned the page with the sysbuild migration.
+  * The :ref:`ug_bt_fast_pair` integration to document integration steps for the FMDN extension, the locator tag use case, and aligned the page with the sysbuild migration.
```

Comment:
Shouldn't update to the following integration guide be also duplicated in the Integrations category as it is done for CoreMark and DULT integration guides?

Reply:
We don't really want to duplicate everything from the Documentation section. In the case of Coremark and DULT integration guides, we wanted to accentuate them as it is the first time they appear in the doc.
The distinction between which doc updates should be put in the Documentation section vs which of them should be put in other, dedicated sections is always quite arbitrary and subjective. We as tech writers always have a dedicated meeting before a release to discuss this. I wouldn't like to restart this discussion.

Reply:
This update was more significant and introduced more content than the brand-new integration guides for CoreMark or DULT:

https://github.com/nrfconnect/sdk-nrf/commit/0567e5906a45ab386704b0d9c9be3c81e47d186a
(535 additions for the FMDN content alone without taking follow-up FMDN updates and sysbuild alignment).

 Perhaps, we should also mention it there.

Reply:
agreed and changed